buildSrc/src/main/kotlin/global-compiler-options.gradle.kts

```diff
@@ -28,7 +37,7 @@ tasks.withType(KotlinCompilationTask::class).configureEach {
             }
 
             allWarningsAsErrors = werrorEnabled
-
+            logger.info("allWarningsAsErrors=$werrorEnabled")
```

Comment:
Is this logging really needed?

Reply:
yes, that will allow us to make sure we use correct settings

Reply:
I mean, that's not exactly what I would want to see in the build log

<img width="415" alt="image" src="https://github.com/user-attachments/assets/9aa4d4de-df84-4a60-9866-fb8a50b1cd19" />

Reply:
Maybe move it to `Added Kotlin compiler flags` section then?

Reply:
```
tasks.withType<KotlinCompilationTask<*>>().configureEach {
    doFirst {
      logger.info("Added Kotlin compiler flags: ${compilerOptions.freeCompilerArgs.get().joinToString(", ")}")
      logger.info("Werror state: ${compilerOptions.allWarningsAsErrors.get()}")
   }
}

Reply:
fair, fixed